plugins/inputs/dmcache/README.md

```diff
@@ -0,0 +1,15 @@
+# DMCache Input Plugin
+
+This plugin provide a native collection for dmsetup based statistics for dm-cache.
+
+This plugin requires sudo, that is why you should setup and be sure that the telegraf is able to execute sudo without a password.
+
+`sudo /sbin/dmsetup status --target cache` is the full command that telegraf will run for debugging purposes.
+
+## Configuration
+
+```
+[[inputs.dmcache]]
+  ## Whether to report per-device stats or not
+  per_device = true
+```
```

Comment:
All of the fields are basically taken from the output of the `dmsetup status --target=cache` output for the given disks. In this case, we want to split all the data into multiple lines per disk we set up.